### DISCUSSION TITLE
Can’t depackage Hyperion on pi 3

### DISCUSSION THREAD

**Discussion by bitterfitter** [hyperion-project/hyperion.ng#1793] [Q&A]
When I try to dpkg Hyperion I get an error about libcec6. 
![IMG_5447](https://github.com/user-attachments/assets/ba3c0fa9-7720-401f-b841-6f66c58c7c0a)

using OS lite 64 bit from the raspberry pi imager.

**Comment by Paulchen-Panther** [comment]
`curl -sSL https://releases.hyperion-project.org/install | bash`

**Reply by bitterfitter** [reply]
Tried that and got this below. 

![image](https://github.com/user-attachments/assets/9db9d2a8-ef31-4602-8a3b-0e031255549a)

**Reply by Paulchen-Panther** [reply]
You must of course uninstall the previous version. 

https://stackoverflow.com/a/48726142/4517819

**Reply by bitterfitter** [reply]
Thank you. 😊 
Just for my knowledge, what is actually causing this?  Is it the OS I am using or the Hyperion?  I would think it is an issue in the OS. If that’s the case, am I really the first person to have this issue with this?  And is there a different OS I could use instead of the 64bit lite version I chose to run my pi for Hyperion?  Just asking for future reference and to help others

**Comment by bitterfitter** [comment]
So if reinstalling Hyperion is the answer, then that must be the issue here and not the OS. Now I start to wonder why I got the same issue even when I tried installing older versions of Hyperion? I wouldn’t think the same issue would be present in every version of Hyperion

**Reply by Paulchen-Panther** [reply]
The problem is simple. You have installed the wrong Hyperion architecture. On your screenshot you can see that you wanted to install the armhf version on an arm64 system. Did your installation work with the repository version?

https://releases.hyperion-project.org

**Reply by bitterfitter** [reply]
I have not had time to work on it further. I am going to jump in again today and work it out. 
is it possible for me to start over and install the OS again and then the correct Hyperion file using the same sd card?

**Reply by Paulchen-Panther** [reply]
You can also use a ready-made Raspberry Pi OS image with pre-installed Hyperion. More details here:
https://docs.hyperion-project.org/user/HyperBian.html#hyperbian

**Reply by bitterfitter** [reply]
That’s great. I had no idea that existed. 
so I downloaded that image and installed it on the same sd card. I seem to be having an ssh issue now. I can’t connect to my pi. 
I have noticed in the past when I attempted this and customized the settings with the imager to set enable ssh and a different username and password, the imager appears to be trying to install a completely different password for my WiFi. I change it back to what I need and save and run the imager but it still doesn’t work. 
any thoughts as this seems to be my last hurdle.
